docs/source/intro.rst

```diff
@@ -31,3 +31,10 @@ These segments are the following:
 We identify these segments by their finishing level, i.e. the Level 1 products come from the Level 0 to Level 1 segment
 which can be called just the Level 1 segment for short. The processing description and code you'll find here is
 organized in this manner.
+
+PUNCH and Python
+----------------
+
+The PUNCH framework is built using Python - an object-oriented language with a large user / code base in astronomy and solar physics. The pipeline and tools for querying / loading PUNCH data use the Python language, along with the SunPy and astroPy software libraries. A number of useful tutorials exist online, including the official `Python tutorial <https://docs.python.org/3/tutorial/index.html>`_ and the `Hitchhiker's Guide to Python <https://docs.python-guide.org>`_.
```

Comment:
I think it's capitalized as "Astropy" instead. After that change, feel free to merge.